modules/eks/cluster/variables.tf

```diff
@@ -93,27 +93,11 @@ variable "cluster_log_retention_period" {
   nullable    = false
 }
 
-variable "apply_config_map_aws_auth" {
-  type        = bool
-  description = "Whether to execute `kubectl apply` to apply the ConfigMap to allow worker nodes to join the EKS cluster"
-  default     = true
-  nullable    = false
-}
-
-variable "map_additional_aws_accounts" {
-  type        = list(string)
-  description = "Additional AWS account numbers to add to `aws-auth` ConfigMap"
-  default     = []
-  nullable    = false
-}
-
-variable "map_additional_worker_roles" {
-  type        = list(string)
-  description = "AWS IAM Role ARNs of worker nodes to add to `aws-auth` ConfigMap"
-  default     = []
-  nullable    = false
-}
-
```

Comment:
The variable `map_additional_worker_roles` is still used in your main.tf, so this needs to be restored